README.md

```diff
@@ -417,6 +417,12 @@ Restricts Pieman to only creating an artifact (a file with the `.mender` extensi
 
 Note, that the parameter conflicts with `CREATE_ONLY_CHROOT` and `ENABLE_MENDER`.
 
+##### ENABLE_BSC_CHANNEL=false
+
+Specifies whether to run the build status codes server (also known as bscd). If the parameter is set to `true`, Pieman will check the connection to the Redis server (specified via `REDIS_HOST` and `REDIS_PORT`) and, in case there is no problem with that, it will be pushing the build status codes to the channel named `bscd-${PROJECT_NAME}`.
```

Comment:
What is 'd' stands for in the 'bscd' abbreviature?

Reply:
daemon